q2_phylogeny/plugin_setup.py

```diff
@@ -476,3 +474,94 @@
                  'methods.'
                  )
 )
+
+plugin.pipelines.register_function(
+    function=q2_phylogeny.align_to_tree_mafft_iqtree,
+    inputs={
+        'sequences': FeatureData[Sequence],
+    },
+    parameters={
+        'n_threads': Int % Range(0, None),
```

Comment:
We now have a slightly cleaner mechanism for doing the "auto" business:

```suggestion
        'n_threads': Int % Range(1, None) | Str % Choices(['auto']),
```

This change relies on https://github.com/qiime2/q2-alignment/pull/65 & https://github.com/qiime2/q2-phylogeny/pull/64

Reply:
Okay great! Done.